Mathlib/GroupTheory/Perm/Sign.lean

```diff
@@ -78,7 +78,7 @@ def swapFactorsAux :
         fun {_} hg => ((List.mem_cons).1 hg).elim (fun h => ⟨x, f x, hfx, h⟩) (m.2.2 _)⟩
 
 /-- `swapFactors` represents a permutation as a product of a list of transpositions.
-The representation is non unique and depends on the linear order structure.
+The representation is nonunique and depends on the linear order structure.
```

Comment:
dito regarding hyphens

Reply:
https://www.merriam-webster.com/dictionary/nonunique

Also general comment below.

Reply:
In the litterature, "non-unique" is more popular than "nonunique", but either option is popular. When a word is listed in a Merriam Webster, the recommendation is also to use the spelling presented there. Hence, I'd prefer not adding a hyphen here.